app/domain/errors.rb

```diff
@@ -20,6 +20,11 @@ module Conjur
       code: "CONJ00046E"
     )
 
+    TrustedProxiesMissing = ::Util::TrackableErrorClass.new(
```

Comment:
looking at the message, it seems that the name is not good enough. how about `TrustedProxiesFetchFailed`?